rtf_vis_tool/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>GTSFM Pipeline Visualizer</title>
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>
```

Comment:
boilerplate html code

Reply:
I don't love the word "pipeline". Maybe we can call it just "GTSFM Computational Graph Visualizer"?